apps/accounts/emails.py

```diff
@@ -1,5 +1,5 @@
 import random
-import logging
+import apps.logging.request_logger as logging
```

Comment:
Do you think we should create constants for these loggers too?
`logger = logging.getLogger('hhs_server.%s' % __name__)`
Like:
`HHS_SERVER = 'hhs_server`

Reply:
it's good to not to use same purpose string literal multiple locations, ...